backends/aptcc/pk-debconf-helper.c

```diff
@@ -0,0 +1,54 @@
+#include "config.h"
+#include <stdio.h>
+#include <packagekit-glib2/pk-client.h>
+#include <packagekit-glib2/pk-client-helper.h>
+#include <systemd/sd-daemon.h>
+
+static GMainLoop *main_loop;
+static PkClientHelper *helper;
+
+gboolean exit_loop(gpointer user_data)
+{
+	g_debug("Checking for active connections");
```

Comment:
This file doesn't match the existing code in having space between the function name and parenthesis.

Reply:
It does match the rest (or most, I did not check it all) of the aptcc backend style, though.